sources/packages/api/src/services/student-file/student-file.service.ts

```diff
@@ -25,7 +25,7 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
     const newFile = new StudentFile();
     newFile.fileName = createFile.fileName;
     newFile.uniqueFileName = createFile.uniqueFileName;
-    newFile.groupName = createFile.groupName;
+    if (createFile.groupName) newFile.groupName = createFile.groupName;

```

Comment:
I would suggest that this should not have a pre-defined value to force the consumer to give some clue about the origin of the temporary file.

Reply:
please refresh the page, reverted the changes